src/networks.json

```diff
@@ -753,6 +753,20 @@
     "start": 366482,
     "logo": "ipfs://QmZLQVsUqHBDXutu6ywTvcYXDZG2xBstMfHkfJSzUNpZsc"
   },
+  "595": {
+    "key": "595",
+    "name": "Mandala TC7",
```

Comment:
What is TC7? i see this `[Acala Mandala Testnet](https://acala.network/)`  on chainlist. maybe we can use same here?

Reply:
TC7 is just the bump version for testnet , should be alright to use `Acala Mandala Testnet` @Jianzhuo